content/docs/fundamentals/converting/resolving-errors@pt_BR.md

```diff
@@ -0,0 +1,350 @@
+---
+$title: Corrigir erros de validação
+$order: 2
+toc: true
+---
+
+[TOC]
+
+Nesta seção, analisaremos e resolveremos os erros de validação de AMP na nossa página AMP.  Esses erros podem aparecer em outra ordem no seu console.
+
+## Incluir charset
+
+Para começar, corrigiremos o seguinte erro:
+
+<pre class="error-text">
+A tag obrigatória "meta charset=utf-8" está ausente ou incorreta.
+</pre>
+
+Para que o texto seja exibido corretamente, é necessário especificar o charset das páginas AMP. As informações do meta charset também precisam ser o primeiro filho da tag `<head> `. Essa tag precisa ser a primeira para evitar a reinterpretação de conteúdo adicionado antes da tag meta charset.
+
+**Adicione** o código a seguir como a primeira linha da tag `<head>`:
+
+
+```html
+<meta charset="utf-8" />
+```
+
+**Salve** o arquivo e atualize a página. Verifique se o erro de charset ainda ocorre.
+
+## Incluir link canônico
+
+Agora, veja este erro:
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).